src/main/java/org/apache/pirk/querier/wideskies/QuerierDriverCLI.java

```diff
@@ -55,7 +58,7 @@
   public static final String PAILLIERBITSIZE = "paillierBitSize";
   public static final String BITSET = "bitSet";
   public static final String CERTAINTY = "certainty";
-  public static final String QUERYNAME = "queryName";
+  public static final String QUERYID = "queryID";
```

Comment:
See previous comment on the QuerierDriver:

The queryName corresponds to the schemaName in the query-schema.xsd file not to the user-given identifier for the query -- the identifier for this query is actually the queryNum. Notice that queryNum is currently a double -- if we want to start appending dates or whatever else to it, we need to change its type to String throughout.